scripts/generate_c_api.py

```diff
@@ -755,6 +755,12 @@ def create_duckdb_c_ext_h(file, ext_api_version, function_groups, api_struct_def
 // Place in global scope of any C/C++ file that needs to access the extension API
 #define DUCKDB_EXTENSION_EXTERN extern {DUCKDB_EXT_API_STRUCT_TYPENAME} {DUCKDB_EXT_API_VAR_NAME};
 
+#ifdef _WIN32
+#define ENTRYPOINT_VISIBILITY __declspec(dllexport)
```

Comment:
```suggestion
#define DUCKDB_CAPI_ENTRY_VISIBILITY __declspec(dllexport)
```
Maybe DUCKDB_CAPI_ENTRY_VISIBILITY or anyhow starting with `DUCKDB_` to avoid clashes?

Reply:
Sure, let's do that